src/index.html

```diff
@@ -7,8 +7,38 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home">
+        <img 
+          class="header__logo" 
+          src="images/logo.png" 
+          alt="Logo of 'MOYO' company"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__ul">
+          <li class="nav__item"><a href="#apple" class="nav__link is-active">Apple</a></li>
```

Comment:
please add right indents for html structure
for example:
```suggestion
          <li class="nav__item">
            <a href="#apple" class="nav__link is-active">
              Apple
            </a>
          </li>
```

Reply:
check it for whole page

Reply:
is not fixed

Reply:
Done